services/movie_session.py

```diff
@@ -1,6 +1,6 @@
 from django.db.models import QuerySet
 
-from db.models import MovieSession
+from db.models import MovieSession, Ticket
```

Comment:
Consider using double quotes for consistency in the import statement, as per the code style checklist.